canonical/ubuntu.com
# WD-5128 - Build the ai/roadshow page

## Done

- Build the ai/roadshow page

## QA

- Open the demo
- Go to /ai/roadshow
- Check the copy matches the [copy doc](https://docs.google.com/document/d/1CA0GUU6yRB65yfDaKi1dgN8DED2bDAFI2h-swSVtMtg/edit)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5128
